src/software/ai/passing/pass_generator_test.cpp

```diff
@@ -122,7 +122,9 @@ TEST_F(PassGeneratorTest, check_pass_converges)
         EXPECT_LE((converged_pass.receiverPoint() - pass.receiverPoint()).length(), 0.3);
         EXPECT_LE(abs(converged_pass.speed() - pass.speed()), 0.3);
         EXPECT_LE(abs((converged_pass.startTime() - pass.startTime()).getSeconds()), 0.2);
+        (void)score;
     }
+    (void)converged_score;
```

Comment:
Is this just ignoring unused variables? Could we perhaps add an `UNUSED` macro?

Reply:
done